From 63f43743ab6adab27e85445338126bee08911ca9 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ahelenia=20Ziemia=C5=84ska?=
 <nabijaczleweli@nabijaczleweli.xyz>
Date: Fri, 12 Jul 2024 19:56:36 +0000
Subject: [PATCH] 16657 fnmatch(3c) typos FNM_CASEFOLD as FNM_FOLDCASE Reviewed
 by: Andy Fiddaman <illumos@fiddaman.net> Reviewed by: Robert Mustacchi
 <rm+illumos@fingolfin.org> Reviewed by: Bill Sommerfeld
 <sommerfeld@hamachi.org> Reviewed by: Dominik Hassler <hadfl@omnios.org>
 Approved by: Patrick Mooney <pmooney@pfmooney.com>

---
 usr/src/man/man3c/fnmatch.3c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/usr/src/man/man3c/fnmatch.3c b/usr/src/man/man3c/fnmatch.3c
index 3d8737aacd..78127f555a 100644
--- a/usr/src/man/man3c/fnmatch.3c
+++ b/usr/src/man/man3c/fnmatch.3c
@@ -44,7 +44,7 @@
 .\" Portions Copyright (c) 2002, Sun Microsystems, Inc.  All Rights Reserved
 .\" Copyright 2017 Nexenta Systems, Inc.
 .\"
-.Dd August 14, 2017
+.Dd July 12, 2024
 .Dt FNMATCH 3C
 .Os
 .Sh NAME
@@ -121,7 +121,7 @@ implementation of
 header
 .In fnmatch.h
 provides the
-.Dv FNM_FOLDCASE
+.Dv FNM_CASEFOLD
 flag having the same meaning.
 .It Dv FNM_NOESCAPE
 If not set, a backslash character
-- 
2.46.0

